bindings/python/tests/conftest.py

```diff
@@ -50,7 +50,10 @@ def setup_config(service_name):
     for key in os.environ.keys():
         if key.lower().startswith(prefix):
             config[key[len(prefix) :].lower()] = os.environ.get(key)
-
+    disable_random_root = False if os.environ.get(
```

Comment:
plz format code by using black or ruff

Reply:
Please don't submit review like this one. It's not contributor's job before we adding it in CI.